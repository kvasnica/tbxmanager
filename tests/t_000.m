function t_000
% tests error messages

onCleanup(@(x) tbx_setupTest('done')); tbx_setupTest('start');

try, tbxmanager install, end
try, tbxmanager uninstall, end
try, tbxmanager enable, end
try, tbxmanager disable, end
try, tbxmanager source, end
try, tbxmanager source add, end
try, tbxmanager source remove, end

end
