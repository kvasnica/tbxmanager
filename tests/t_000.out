--- tbxmanager install 

Error using ==> tbxmanager>validate_notempty at 558
Name of a toolbox must be provided.

--- tbxmanager uninstall 

Error using ==> tbxmanager>validate_notempty at 558
Name of a toolbox must be provided.

--- tbxmanager enable 

Error using ==> tbxmanager>validate_notempty at 558
Name of a toolbox must be provided.

--- tbxmanager disable 

Error using ==> tbxmanager>validate_notempty at 558
Name of a toolbox must be provided.

--- tbxmanager source 

Error using ==> tbxmanager>main_source at 263
The "source" command requires two additional inputs.

--- tbxmanager source add 

Error using ==> tbxmanager>main_source at 263
The "source" command requires two additional inputs.

--- tbxmanager source remove 

Error using ==> tbxmanager>main_source at 263
The "source" command requires two additional inputs.
